### PR TITLE
[devtools-translate] Chinese - What's New in DevTools (Chrome 103)

### PR DESCRIPTION
**Pull request by liuliangsir**
Fixes: https://github.com/GoogleChrome/developer.chrome.com/issues/3018

This PR adds Simplified Chinese translations for "What's New In DevTools (Chrome 103)". https://developer.chrome.com/blog/new-in-devtools-103/

### CODE REVIEW

File: site/zh/blog/new-in-devtools-103/index.md [hunk 1]
```diff
@@ -30,154 +29,204 @@ draft: true
...
+
 <!-- Here are a few fixes on sourcemaps to improve the overall debugging experience: -->
- 
+这里有几个对 sourcemaps 的改进，旨在提高整体调试体验：
+
 <!-- - Correct mapping between location and offset for inline scripts and source location
 - Use fallback information for frame’s text location
 - Properly resolve relative urls with frame's URL   -->
- 
+- 针对 inline scripts 和 source location，修正位置跟偏移之间的映射关系
```
Comment (hanselfmu) [inline site/zh/blog/new-in-devtools-103/index.md:221]: Should we translate "inline scripts" and "source location" here? I think these terms are common enough to be translated.

File: site/zh/blog/new-in-devtools-103/index.md [hunk 2]
```diff
@@ -30,154 +29,204 @@ draft: true
...
 
 
 <!-- ## Reorder panes in the Elements panel {: #reorder-pane } -->
+## 重新排列 Elements 面板中的窗格 {: #reorder-pane }
 
 <!-- You can now reorder panes in the **Elements** panel based on your preference. -->
+您现在可以根据您的喜好来重新排列 Elements 面板中的窗格。
 
 <!-- For example, when you open DevTools on a narrow screen, the [Accessibility](/docs/devtools/accessibility/reference/#pane) pane is hidden under the **Show more** button. If you frequently debug accessibility issues, you can now drag the pane to the front for easier access. -->
+例如，当您在窄屏上打开 DevTools，[Accessibility](/docs/devtools/accessibility/reference/#pane) 窗格会被隐藏在 **Show more** 按钮下。如果您需要经常调试 Accessibility 问题，那么您现在可以将 Accessibility 窗格拖到前面，以便于更容易访问。
```
Comment (hanselfmu) [inline site/zh/blog/new-in-devtools-103/index.md:127]: I think we have translated "Show more" to "展开" in DevTools.

**Review by hanselfmu — APPROVED** [review]

File: site/zh/blog/new-in-devtools-103/index.md (hunk 2 above)
Comment (liuliangsir) [reply site/zh/blog/new-in-devtools-103/index.md:127]: okay

File: site/zh/blog/new-in-devtools-103/index.md (hunk 1 above)
Comment (liuliangsir) [reply site/zh/blog/new-in-devtools-103/index.md:221]: like 行内脚本和源码位置？

Comment (jecfish) [inline site/zh/blog/new-in-devtools-103/index.md:221]:
```suggestion
- 针对行内脚本（inline scripts）和源码位置（source location），修正位置跟偏移之间的映射关系
```

**Review by jecfish — APPROVED** [review]

**Pull request merged**